extension/js/common/composer/composer-quote.ts

```diff
@@ -0,0 +1,165 @@
+/* © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { Composer } from '../composer.js';
+import { MessageToReplyOrForward, ComposerUrlParams } from './interfaces/composer-types.js';
+import { Xss } from '../platform/xss.js';
+import { Str } from '../core/common.js';
+import { Ui } from '../browser.js';
+import { Api, ProgressCb } from '../api/api.js';
+import { Catch } from '../platform/catch.js';
+import { ComposerComponent } from './interfaces/comopser-component.js';
+import { Google } from '../api/google.js';
+import { Mime } from '../core/mime.js';
+import { Buf } from '../core/buf.js';
+import { FormatError, PgpMsg } from '../core/pgp.js';
+import { BrowserMsg, Bm } from '../extension.js';
+import { Store } from '../platform/store.js';
+
+export class ComposerQuote extends ComposerComponent {
+    private messageToReplyOrForward: MessageToReplyOrForward | undefined;
+    private msgExpandingHTMLPart: string | undefined;
+
+    get expandingHTMLPart(): string | undefined {
+        return this.msgExpandingHTMLPart;
+    }
+
+    constructor(composer: Composer, urlParams: ComposerUrlParams) {
```

Comment:
Maybe this constructor is unnecessary? If if you want to keep it in for code clarity, you can.

Reply:
Agree, this one is unnecessary